### PR TITLE
Python Sliding Window Solution

### PR DESCRIPTION
**Pull request by dsmyda**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/4_Gold/Sliding_Window.mdx [hunk 1]
```diff
@@ -264,6 +264,29 @@ static ArrayList<Integer> maxSlidingWindow(int[] nums, int k) {
 
 </JavaSection>
 
+<PySection>
+
+```python
+def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
+	d = collections.deque()
+	
+	def enqueue(i):
```
Comment (SansPapyrus683) [inline content/4_Gold/Sliding_Window.mdx:273]: type annotations for this function as well

**Review by SansPapyrus683 — APPROVED** [review]
just 1 thing

File: content/4_Gold/Sliding_Window.mdx (hunk 1 above)
Comment (dsmyda) [reply content/4_Gold/Sliding_Window.mdx]: Thanks for taking a look. I just added type annotations.

File: content/4_Gold/Sliding_Window.mdx [hunk 2]
```diff
@@ -264,6 +264,29 @@ static ArrayList<Integer> maxSlidingWindow(int[] nums, int k) {
 
 </JavaSection>
 
+<PySection>
+
+```python
```
Comment (sathu-beep) [inline content/4_Gold/Sliding_Window.mdx:269]:
```suggestion
```Python
```

Comment (SansPapyrus683) [reply content/4_Gold/Sliding_Window.mdx:269]: what

**Pull request merged**